### ISSUE TITLE
Raspberry Pi Zero disconnection

### ISSUE THREAD

**Issue by junior-senior** [dzhu/myo-raw#27]
After approximately 2-3 seconds after the program starts printing emg data onto IDLE/terminal the armband disconnects and then the emg data is no longer printed. Just wondering if anyone else has this problem and knows of a fix.

**Comment by hananabilabd** [comment]
@junior-senior  did you get to know the answer ??

**Comment by junior-senior** [comment]
@hananabilabd I did not unfortunately. It worked fine on a Pi 3 so opted for that instead.

**Comment by hananabilabd** [comment]
@junior-senior  yeah um working also on Pi 3 but I have an issue of the slow rate of EMG Raw data 
it gives me 512 data frame every 10 sec which is so slow 
<<Do you faced the same issue ??

**Comment by PerlinWarp** [comment]
Hey @junior-senior , If you are using Windows the issue was the Myo timing out, I made a version of the driver without this problem [here](https://github.com/PerlinWarp/pyomyo). My fix for timeout on Windows is in [this commit](https://github.com/PerlinWarp/pyomyo/commit/189b08f62fcca5b4f880d7f1565700a57ee75568).
